org.signal.Signal.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: org.signal.Signal
```

Comment:
this app already exist.

Reply:
Yes, but the beta branch has been unmaintained for a few years, so I would like to take responsibility for it.

Reply:
then submit a PR on said repository

Reply:
Ok, will close this PR